packages/portalnetwork/src/networks/history/util.ts

```diff
@@ -294,13 +295,60 @@ export const verifyPreCapellaHeaderProof = (
     'blockHash',
   ])
   const reconstructedBlock = ssz.bellatrix.BeaconBlock.createFromProof({
-    witnesses: proof.beaconBlockHeaderProof,
+    witnesses: proof.beaconBlockProof,
     type: ProofType.single,
     gindex: elBlockHashPath.gindex,
     leaf: elBlockHash,
   })
 
-  if (!equalsBytes(reconstructedBlock.hashTreeRoot(), proof.beaconBlockHeaderRoot)) return false
+  if (!equalsBytes(reconstructedBlock.hashTreeRoot(), proof.beaconBlockRoot)) return false
+  return true
+}
+
+export const verifyPostCapellaHeaderProof = (
+  proof: ValueOfFields<{
+    beaconBlockProof: VectorCompositeType<ByteVectorType>
```

Comment:
i think you could simplify this to Uint8Array[]

Reply:
I'm not sure what you mean though I just realized I have the wrong type here since this is a list now. How would I switch it to a `Uint8Array[]`?